dsl/src/main/scala/com.crobox.clickhouse/dsl/TableColumn.scala

```diff
@@ -49,6 +49,14 @@ abstract class TypeCastColumn[V](targetColumn: AnyTableColumn) extends Expressio
 case class Reinterpret[V](typeCastColumn: TypeCastColumn[V] with Reinterpretable)
     extends TypeCastColumn[V](typeCastColumn)
 
+
+//FIXME: Here until we add them to the DSL trough the vocabulary update (feature/voc-typed-magnets)
+abstract class ArithmeticFunction[V]() extends ExpressionColumn[V](EmptyColumn())
+case class Multiply[V](left: AnyTableColumn, right: AnyTableColumn) extends ArithmeticFunction[V]
```

Comment:
Shouldn't left and right be restricted to numeric?

Reply:
you can also work on dates, but we could skip that for now.. for me this is really a quick and dirty setup. 

Want me to change it?

Reply:
Good for dictionary update